backend/app/core/config.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import yaml
+from pydantic import BaseSettings, RedisDsn, validator
+
+
+class RedisSettings(BaseSettings):
+    uri: RedisDsn
+
+
+class ServerSessionsSettings(BaseSettings):
+    session_key: str
+    expires_in: int
+
+
+class Settings(BaseSettings):
+
+    redis: RedisSettings
+    server_sessions: ServerSessionsSettings
+
+    @classmethod
+    def from_yaml(cls, path_to_file):
+        absolute_path_to_file = Path().absolute() / path_to_file
+
+        try:
+            with open(str(absolute_path_to_file)) as f:
```

Comment:
You don't need to cast to str here iirc Python 3.7+ supports Path as an argument to open.